doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -185,14 +168,121 @@ Traceability to Reference Model.
 | `req.sec.zon.04` | Zoning | The Architecture **must** support integration with Corporate Identity Management systems. | |
 | `req.sec.cmp.02` | Compliance | The Architecture **must** comply with all applicable standards and regulations. | |
 | `req.sec.cmp.03` | Compliance | The Architecture **must** comply with all applicable regional standards and regulations. | |
-<!-- | `req.sec.ntw.01` | Networking | The Architecture **must** have the underlay network include strong access controls that comply with the applicable security standard (national, regional), for example ISO27001. | |
-| `req.sec.ntw.02` | Networking | The Architecture **must** have all security logs stored in accordance with applicable security standard (national, regional), for example ISO27001. | | -->
 | `req.sec.ntw.03` | Networking | The Architecture **must** have the underlay network incorporate encrypted and/or private communications channels to ensure its security. | |
-|----|--------------|-----------------------------|-------|
 | `req.sec.ntw.04` | Networking | The Architecture **must** configure all of the underlay network components to ensure the complete separation from the overlay customer deployments. | |
 | `req.sec.ntw.05` | Networking | The Architecture **must** have the underlay network include strong access controls that adhere to the V1.1 NIST Cybersecurity Framework. | |
+<!-- | `req.sec.ntw.01` | Networking | The Architecture **must** have the underlay network include strong access controls that comply with the applicable security standard (national, regional), for example ISO27001. | |
+| `req.sec.ntw.02` | Networking | The Architecture **must** have all security logs stored in accordance with applicable security standard (national, regional), for example ISO27001. | | -->
+
 <p align="center"><b>Table 2-8:</b> OpenStack Security Requirements.</p>
 
+
+
+<a name="2.4"></a>
+### 2.4 Architecture and OpenStack Recommendations
+
+The requirements listed in this section are optional, viz. do not need to be implemented in conformant implementations.
+
+<a name="2.4.1"></a>
+2.4.1 General Recommendations
+
+| Ref # | sub-category | Description |  Traceability |
+|----|-------|---------------------|-------------|
+| `rec.gen.cnt.01` | Cloud nativeness | The Architecture **should** consist of stateless service components. However, where state is required it must be kept external to the component. | OpenStack consists of both stateless and stateful services where the stateful services utilize a database. For latter see "[Configuring the stateful services](https://docs.openstack.org/ha-guide/control-plane-stateful.html)"|
```

Comment:
I am a little concerned about how close 'rec.gen.*' is to 'req.gen.*' and the potential for inadvertent mistakes.  Can we use just one, req.gen?
* no inadvertent lexicographical mistakes
* no embedding of requirement strength ('rec'comendation vs 'req'uirement) in the naming, so if we change a requirement from a must to a should, or vice versa, we don't need to renumber?

Reply:
@karinesevilla had suggested using "rec". @kedmison suggested using "req" labeling even for Recommendations as was originally the case. 

In addition to advantages stated by @kedmison, using "req" has the advantage that none of the other RA-1 chapters have to edited. 

So naming has been reverted to use of "req"